arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -258,7 +227,10 @@
             "Inherit",
             "Custom"
           ],
-          "type": "string"
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AutomaticTuningMode"
```

Comment:
ModelAsString is set in code to false, but it's not visible in swagger. It looks like swagger generator generates modelAsString only in cases when it's set to true. I will start a thread with Nathan to discuss this issue